package.json

```diff
@@ -17,7 +17,9 @@
     "test:routes": "BABEL_DISABLE_CACHE=1 HTTP_PROXY= HTTPS_PROXY= TZ=US/Pacific mocha --env=environments/test.env --recursive --require babel-core/register --require babel-polyfill --compilers js:babel-register ./test/setup.js ./test/routes/**/*",
     "migrate:latest": "knex migrate:latest",
     "migrate:rollback": "knex migrate:rollback",
-    "generate-badges": "node scripts/badges.js"
+    "shonkwrap": "shonkwrap",
+    "generate-badges": "node scripts/badges.js",
+    "poll-repositories": "babel --out-dir=dist src scripts/poller.js --copy-files 2>&1 > /dev/null && node dist/scripts/poller.js"
```

Comment:
Good point, I've create a convenient (quicker) `npm run build:scripts` with that so one can run `npm run build:scripts 2>&1 > /dev/null && npm run poll-repositories -- --env=environments/dev.env` and skip the "webpack" part of the default build process.